style.css

```diff
@@ -68,46 +65,337 @@
     line-height: 24px;
     color: #344563;
     margin-bottom: 10px;
-    padding-left: 20px;
   }
 
   .social-media {
     display: flex;
     align-items: center;
-    gap: 10px;
+    gap: 18px;
+  }
+
+  .link-text a {
+    text-decoration: none;
   }
 
   .social-media li {
     list-style: none;
-    padding-left: 20px;
   }
 
   .social-media li a {
     text-decoration: none;
     color: #505f79;
   }
 
+  .language-set a img {
+    padding-left: 20px;
+    margin-top: 10px;
+  }
+
   .social-media li a img {
     width: 20px;
   }
 
-  .btn {
+  .link-text {
     font-size: 17px;
     font-weight: 500;
-    text-align: center;
-    padding: 15px;
     width: 200px;
     color: #172b4d;
     margin-bottom: 10px;
     cursor: pointer;
     transition: 0.3;
-    margin-left: 20px;
   }
 
-  .btn:hover {
-    background-color: #5e6c84;
+  /* Works Section */
+
+  .works-section {
+    display: grid;
+    grid-template-rows: repeat(4);
+    padding: 70px 24px;
+    gap: 86px;
+    margin-top: 100px;
+  }
+
+  .work-card {
+    border: 1px solid #dfe1e6;
+    border-radius: 25px;
+    display: flex;
```

Comment:
- [ ] You did well 😃 at adding the border-radius as stated by the previous reviewer, but I see that you put a value of border-radius 25px that's bigger than the Figma design👌, so you can check it from the Figma it will like 16px, so replace it with the correct one.

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/212342211-8ccf056b-d6aa-4aef-870e-e5cb8aa3f653.png)|    ![image](https://user-images.githubusercontent.com/83360636/212342078-2ebbf7d2-e50d-4640-892d-782ae399b1b3.png)|